dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/BaseClasses/BasePVRemote.mo

```diff
@@ -19,6 +19,8 @@ partial model BasePVRemote "Base dynamic model for a remote voltage regulation"
 
   parameter Types.VoltageModule URef0 "Start value of the voltage regulation set point in kV";
   parameter Types.VoltageModule URegulated0 "Start value of the regulated voltage in kV";
+  parameter Real UDeadBandPu(min = 0) "Voltage deadband around the target in pu (base UNom)";
```

Comment:
Could you modify the type in all models where you define these two parameters?
```suggestion
  parameter Types.VoltageModulePu UDeadBandPu(min = 0) "Voltage deadband around the target in pu (base UNom)";
```

Reply:
Done